desktop/apps/artwork/components/auction/templates/bid.jade

```diff
@@ -35,12 +35,12 @@ if showAuctionForm
             option( value=val data-display=display )= display
 
     - bidAmount = 'Bid ' + accounting.formatMoney(firstIncr / 100, sale_artwork.symbol, 0)
-    
+
     case bidderStatus
       when 'registration-pending'
         .artwork-auction__bid-form__button.avant-garde-button-black.is-block.is-disabled
           | Registration Pending
-      when 'registration-required'
+      when 'approval-required'
```

Comment:
Okay! I've updated this since I now understand the case here (different from the auction page) where we care that the auction _may_ require extra approval-- it's so we redirect to the plain auction registration form instead of the combo bid form after you click the button. (just reiterating what you had before).

I updated the naming here since I think that was the source of my confusion. 😄